capa/main.py

```diff
@@ -526,6 +596,14 @@ def main(argv=None):
             choices=(BACKEND_VIV, BACKEND_SMDA),
             default=BACKEND_VIV,
         )
+    parser.add_argument(
+        "--signature",
+        action="append",
+        dest="signatures",
+        type=str,
+        default=[],
+        help="use the given signatures to identify library functions, file system paths to .sig/.pat files.",
+    )
```

Comment:
TODO: need to pull in default signature set, which i suggest we store in subdirectory `./sigs/` and is maybe a submodule like `capa-testfiles`.